internal/discovery/host.go

```diff
@@ -93,6 +99,20 @@ func getHostIPAddresses() ([]string, error) {
 	return ipAddrList, nil
 }
 
+func getHostFQDN() *string {
```

Comment:
Well done! It makes sense actually discovering this in the discovery.
Yes, it did work getting it at config build time, but this sounds more appropriate :+1:

Reply:
Yeah in the end I just went straight to the point mainly because the FQDN can actually change and we'd lose that modification if such an information was collected at the start